src/psyclone/psyGen.py

```diff
@@ -3212,11 +3231,28 @@ class Loop(Node):
     :param valid_loop_types: a list of loop types that are specific \
         to a particular API.
     :type valid_loop_types: list of str
+    :param str annotation: Tag that provides additional information about the \
```

Comment:
Shouldn't this now be a list of annotations and shouldn't annotation be used instead of 'tag'?

Reply:
Yes, better thanks.